cpp/dolfinx/fem/DirichletBC.h

```diff
@@ -132,6 +132,71 @@ template <typename T>
 class DirichletBC
 {
 private:
+  /// @brief Build list of cells (local to process, including ghosts)
+  /// which contain any of the input degrees of freedom.
+  ///
+  /// Given a function space `V` and a set of `dofs`, return cell `c` if
```

Comment:
The function arguments give the type, so no need to repeat. Can this text be removed and covered by the parameter and return docstrings?